scipy/interpolate/tests/test_rbf.py

```diff
@@ -146,5 +146,15 @@ def test_rbf_epsilon_none():
     rbf = Rbf(x, y, epsilon=None)
 
 
+def test_rbf_epsilon_none_collinear():
+    # Check that collinear points in one dimension doesn't cause an error
+    # due to epsilon = 0
+    x = [1, 2, 3]
+    y = [4, 4, 4]
+    z = [5, 6, 7]
+    rbf = Rbf(x, y, z, epsilon=None)
+    assert_almost_equal(rbf.epsilon, 2.0/3)
```

Comment:
This is too strict IMO. 2/3 is an implementation detail and I don't think we should insist on this particular estimate of the default epsilon. I'd just test that epsion is non-zero, e.g., `assert_(rbf.epsilon > 0)`.